student.rb

```diff
@@ -1,12 +1,25 @@
+require_relative 'person'
+require_relative 'classroom'
+
 class Student < Person
-  attr_accessor :classroom
+  attr_accessor :name
```

Comment:
:red_circle: The superclass Person already has a name property. You can remove this line of code and use the name property from the Person class instead. This will avoid duplication of code.